script.js

```diff
@@ -1,4 +1,159 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+var customerName = prompt("Welcome to a game of Scissors, Paper, Stone! Please enter your name", "You name goes here!");
+var gameMode = prompt(
+  "Welcome to a game of Scissors, Paper, Stone! Please enter your game mode, i.e og, reversed, korean"
+);
+
+// total number of wins for player, computer
+var userScoreCount = 0;
+var comScoreCount = 0;
+var winner = '';
+
+// SPS
+var computerRandom = function() {
+  var ranNum = Math.floor(Math.random() * 3);
+  console.log(ranNum)
+  if (ranNum == 0) {
+    return 'stone';
+  } else if(ranNum == 1) {
+    return 'paper' ;
+  }
+  return 'scissors';
+}
+
+var main = function(input) {
+  var userName = customerName;
+  var computerChoice = computerRandom(); // calling a function
+  
+  if (gameMode == "og") {
+    return og(input)
+  }
+  if (gameMode == "reversed") {
+    return reversed(input)
+  }
+  if (gameMode == "korean") {
+    return korean(input)
+  } 
+  return 'Invalid response. Please only type "og", "reversed", "korean"!'
+}
```

Comment:
Very very clean main function. Good job in abstracting each game mode into a function